keps/2724-topology-aware-scheduling/README.md

```diff
@@ -478,6 +485,14 @@ type PodSetTopologyRequest struct {
   // +optional
   Preferred *string `json:"preferred,omitempty"`
 
+  // Unconstrained indicates the topology assignment for the PodSet should be,
```

Comment:
```suggestion
  // Unconstrained indicates the topology assignment for the PodSet should be,
```
Should be sounds like it is not necessarily. Can we rephrase as  "is" or "is by default"?

Reply:
Done